Sources/NostrSDK/Filter.swift

```diff
@@ -28,6 +28,11 @@ public struct Filter: Codable {
         case limit = "limit"
     }
     
+    /// Creates the filter with the specified parameters
+    ///
+    /// - Parameters:
+    ///   - ids: a list of event ids or prefixes
```

Comment:
It's clear that the ids and authors should be the hexadecimal representations when you read https://github.com/nostr-protocol/nips/blob/master/01.md but I think it would be valuable to also mention it in our public docs. Otherwise, some may accidentally pass in https://github.com/nostr-protocol/nips/blob/master/19.md bech32 encoded identifiers. Non-blocking, but further down, it would be interesting to explore what kind of validations we want in place to just prevent that entire class of problems.